src/screens/Onboarding/steps/deviceSelection.tsx

```diff
@@ -3,27 +3,41 @@ import { useTranslation } from "react-i18next";
 import { useTheme } from "styled-components/native";
 import { useNavigation } from "@react-navigation/native";
 import { TrackScreen } from "../../../analytics";
-import nanoS from "../assets/nanoS";
-import nanoSP from "../assets/nanoSP";
-import nanoX from "../assets/nanoX";
-import { ScreenName } from "../../../const";
+import nanoSSvg from "../assets/nanoS";
+import nanoSPSvg from "../assets/nanoSP";
+import nanoXSvg from "../assets/nanoX";
+import { ScreenName, NavigatorName } from "../../../const";
 import OnboardingView from "../OnboardingView";
 import StyledStatusBar from "../../../components/StyledStatusBar";
 import ChoiceCard from "../../../components/ChoiceCard";
 
-const devices = [nanoX, nanoSP, nanoS];
+const nanoX = { SvgDevice: nanoXSvg, id: "nanoX" };
+const nanoS = { SvgDevice: nanoSSvg, id: "nanoS" };
+const nanoSP = { SvgDevice: nanoSPSvg, id: "nanoSP" };
+const nanoDev = { SvgDevice: nanoXSvg, id: "nanoDev" };
+
+const devices = [nanoX, nanoSP, nanoS, nanoDev];
 
 function OnboardingStepDeviceSelection() {
   const navigation = useNavigation();
   const { t } = useTranslation();
   const { colors } = useTheme();
 
   const next = (deviceModelId: string) => {
-    // TODO: FIX @react-navigation/native using Typescript
-    // @ts-ignore next-line
-    navigation.navigate(ScreenName.OnboardingUseCase, {
-      deviceModelId,
-    });
+    if (deviceModelId === "nanoDev") {
+      // TODO: fix navigation typescript by adding a type def RootStackParamList to the createStackNavigator
```

Comment:
Fixing it means to type i would say at least [BaseOnboardingNavigator](https://github.com/LedgerHQ/ledger-live-mobile/blob/81e9c4873b6028ed17896e742598dfbdf75691c6/src/components/RootNavigator/BaseOnboardingNavigator.js#L48) (and probably [BaseNavigator](https://github.com/LedgerHQ/ledger-live-mobile/blob/81e9c4873b6028ed17896e742598dfbdf75691c6/src/components/RootNavigator/BaseNavigator.tsx#L82), but not sure of the current type of navigation because we use `useNavigation()` just above) like i did with [SyncOnboardingNavigator](https://github.com/LedgerHQ/ledger-live-mobile/blob/81e9c4873b6028ed17896e742598dfbdf75691c6/src/components/RootNavigator/SyncOnboardingNavigator.tsx#L10)

Reply:
So it would probably be better to do it in another PR that add stuff to this one IMO